requirements-dev-lite.txt

```diff
@@ -2,7 +2,7 @@ boto3==1.17.106 # This should match the version in constraints-dev.txt
 flask>=1.0.0 # for s3 test only (with moto)
 freezegun>=0.3.15
 mock-alchemy>=0.2.5
-moto>=1.3.7,<2.0.0
+moto>=2.0.0,<3.0.0
```

Comment:
@lccasagrande Has this version change caused any test failures in other Great Expectations components?  Thank you. /cc @cdkini

Reply:
It did not =D